config/routes.rb

```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  get "static_pages/home"
+  get "static_pages/help"
+  root "static_pages#home"
```

Comment:
plz arrange line code same bellow:
first line: `root "static_pages#home"`
then, After this line, have 1 empty line

then, `get "static_pages/home"`
` get "static_pages/help"`

Reply:
done

Reply:
done